exercises/section3/promise-all/third.js

```diff
@@ -22,6 +22,7 @@ async function handler() {
 
   const items = Array.from(Array(100))
 
+  const pMap = await import('p-map')
```

Comment:
let's make sure this works fine with the minimum Node version required in the workshop. Will also mention @mcollina to make sure this doesn't deviate from the intent of the workshop, otherwise we could just as well stay with the previous version of p-map.